src/app/ubs-admin/components/ubs-sidebar/ubs-sidebar.component.html

```diff
@@ -1,38 +1,24 @@
-<!-- for Header -->
-<div class="wrapper">
-  <div class="logo">
-    <img src="assets/admin-header/logo-admin.svg" />
-  </div>
-  <div class="icons">
-    <img src="assets/admin-header/first-icon-admin.svg" />
-    <img src="assets/admin-header/bell-icon-admin.svg" />
-    <span>En</span>
-    <img src="assets/img/arrow_down.svg" />
-    <a href="/">Home</a>
-    <span [ngStyle]="{ color: 'var(--primary-green)', fontFamily: 'var(--primary-font)' }">Jana Done</span>
-  </div>
-</div>
 <div class="sidebar-icons" #sideBarIcons>
+  <button class="sidebar-toggler" type="button" (click)="toggleSideBar()" #sidebarToggler [disabled]="stopClick"></button>
```

Comment:
can we make it visible ?
![Screenshot 2021-06-08 at 20 08 26](https://user-images.githubusercontent.com/46226026/121228544-8e6b2180-c895-11eb-9adf-ab3c1701c3b4.png)

Reply:
the issue now is that the button is not visible to the user